### PR TITLE
Doc rewrite

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by ericdill** [comment]
What is the best way to do this?  Should I just merge this and then people can propose edits via PR if there is any contention?

**Comment by danielballan** [comment]
Let's let it sit for a week so it is easy for people to leave line comments.

**Comment by tacaswell** [comment]
The docs also do not currently build, that should be fixed before it is
merged.

**Comment by ericdill** [comment]
Am I remembering correctly that we decided to move this to the top-level scikit-beam organization docs?

**Comment by ericdill** [comment]
cc @tacaswell @danielballan

File: doc/index.rst [hunk 1]
```diff
@@ -1,41 +1,149 @@
...
 
+The :ref:`vision<vision>` of `scikit-beam` is to provide simple
+functions useful for the X-ray, Neutron and Electron communities.  The
+primary goal of the scikit-beam project is to provide a centralized
+repository for algorithms that are used in these three scientific
+domains.  `scikit-beam` functions accept and return standard Python
+and numpy datatypes, so they integrate well with other packages from
+the scientific Python community.  Further, the modular design of
+scikit-beam allows its components to be easily reused in ways not
+envisioned by the authors.
```
Comment (tacaswell) [inline doc/index.rst]: @chrisvam Comments on this paragraph?

**Comment by tacaswell** [comment]
@chrisvam The inventory of what we have implemented has been side-tracked my making sure the docs build with out errors.

File: doc/index.rst (hunk 1 above)
Comment (chrisvam) [inline doc/index.rst]: Looks good.  My only thought is that we do have classes that maintain state (for performance) so maybe "functions" should be changed to "functions/classes"?

**Comment by chrisvam** [comment]
Hi Tom,

No worries about the delay.  You and Dan should have received a link to a google-doc (that you can edit) that is my zeroth-order suggestion for what to send to the silx people.  In a nutshell:  I think we should see if we can agree on a hopefully-simple list of requirements for software we can all work on.  And I believe skbeam fulfills the ones I have written down so far.

**Comment by danielballan** [comment]
Normal Travis builds passed. Only the doc build failed, on warning-as-error from sphinx. Merging as is and basing further work on master.

**Pull request merged**